Dungeoneer/src/com/interrupt/dungeoneer/game/Game.java

```diff
@@ -93,8 +94,8 @@
 	public static Tooltip tooltip = new Tooltip();
     public static Stage ui;
 
-    public static Hotbar hotbar = new Hotbar(6,1,0);
-    public static Hotbar bag = new Hotbar(6,3,6);
+    public static Hotbar hotbar;
+    public static Hotbar bag;
```

Comment:
Instead of these being statically accessible from here, it would be better to hold a static reference to the HudManager. That would better encapsulate all of this functionality, that would also make it easier for HUD rendering to also move into the HudManager class.

Reply:
Ah yes, understand. Good point.

Reply:
Updated.